src/scss/header.scss

```diff
@@ -10,5 +10,31 @@ header {
         img{
             max-height: 5em;
         }
+
+    ul {
+        display: flex;
+		justify-content: space-evenly;
+        width: 100%;
+
+        li {
+            list-style: none;
+            
+            a {
```

Comment:
Er staat geen transition op de a, misschien is dit een idee om toe te voegen. De background change van de hover ziet er dan mooier uit.

Reply:
goed idee, ga ik naar kijken